src/frontend/komponenter/Behandling/VedtakOgBeregning/VedtaksresultatSwitch.tsx

```diff
@@ -86,13 +151,150 @@ const VedtaksresultatSwitch: React.FC<Props> = (props: Props) => {
             });
     };
 
+    const VelgAktivitetsplikt = (
```

Comment:
Har ikke sett nøye etter, men er dette verdt en egen komponent?

Reply:
^ enig! Synes at man kan trekke ut litt kode her. Ev. trekke ut `<option>..` koden for litt bedre lesbarhet